### PR TITLE
docs/osf-trivia-list: fix 404 issues

### PR DESCRIPTION
**Pull request by pietrushnic**

### CODE REVIEW

File: docs/osf-trivia-list/deployment.md [hunk 1]
```diff
@@ -154,7 +154,7 @@ Note: flashrom can never write if the flash chip isn't found automatically.
 
 that means your flashrom version is incorrect. Follow the procedure for
 building the right flashrom is described in `Build flashrom` section in the
-[Initial deployment documentation](initial-deployment.md#initial-deployment-manually).
+[Initial deployment documentation](#how-to-install-dasharo-flashrom-fork).
```
Comment (miczyg1) [inline docs/osf-trivia-list/deployment.md:157]: This link does not exist

**Review by miczyg1 — CHANGES_REQUESTED** [review]

File: docs/osf-trivia-list/deployment.md (hunk 1 above)
Comment (pietrushnic) [reply docs/osf-trivia-list/deployment.md:157]: @miczyg1 true, it depends on https://github.com/Dasharo/docs/pull/513

Comment (miczyg1) [reply docs/osf-trivia-list/deployment.md:157]: Maybe worth cascading the PRs? or separate it

**Comment by miczyg1** [comment]
Merged as https://github.com/Dasharo/docs/pull/513

**Pull request closed without merging**